backend/validators/web.py

```
@@ -0,0 +1,76 @@
+
+from pathlib import Path
+from copy import deepcopy
+
+from .base import *
+
+
+class WebModule:
+    _process: asyncio.subprocess.Process | None
+
+    def __init__(self):
+        self.address = f"127.0.0.1:3031"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)